tests/Feature/ArtworkTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Tests\Feature;
+
+use Aic\Hub\Foundation\Testing\FeatureTestCase as BaseTestCase;
+use App\Models\Api\Artwork;
+use App\Models\Article;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\MockApi;
+
+class ArtworkTest extends BaseTestCase
+{
+    use MockApi, WithFaker;
+
+    protected $seed = true;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        // Artworks require at least six default related items.
+        // See `App\Models\Behaviors\HasFeatureRelated::targetItemCount`.
+        Article::factory()->count(6)->published()->visible()->notUnlisted()->create();
+    }
+
+    public function test_artwork_show_redirects_to_canonical_url_when_missing_slug()
+    {
+        $artwork = Artwork::factory()->make();
+        $this->addMockApiResponses($this->mockApiModelReponse($artwork));
+
+        $response = $this->get("/artworks/{$artwork->id}/");
+        $response->assertRedirect("/artworks/{$artwork->id}/{$artwork->titleSlug}/");
```

Comment:
Ideally, we'd want to strip the trailing slash off this URL. I see both URLs work on our website currently:

https://www.artic.edu/artworks/28560/the-bedroom
https://www.artic.edu/artworks/28560/the-bedroom/

Can you remove the trailing slash from these tests and we'll address the trailing slash separately?

Reply:
Done.

I believe adjusting the trailing slashes would be as simple as removing the `$uri/` from our nginx config:
https://github.com/art-institute-of-chicago/utils/blob/main/deploy-provision/roles/base/php-nginx/templates/aic.d/php-locations.conf#L20

Reply:
Great. I created WEB-2560 to track the config update.